### PR TITLE
Pin dandischema to compatible 0.minor version

### PR DESCRIPTION
**Pull request by yarikoptic**
since we are not at luxury to "redeploy" dandi-cli rapidly (installed by
users), I see no other way than this.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -30,7 +30,7 @@ install_requires =
     appdirs
     click
     click-didyoumean
-    dandischema
+    dandischema ~= 0.1.0
```
Comment (jwodder) [inline setup.cfg:33]: This is equivalent to `>= 0.1.0, <0.2.0`.  Is that what you meant, or did you mean `~= 0.1` (which is `>= 0.1, <1.0`)?

Comment (yarikoptic) [reply setup.cfg:33]: 0.2.0 broke it.  so I guess we will need to resort for now for this level.

Comment (yarikoptic) [reply setup.cfg:33]: but I filed https://github.com/dandi/dandischema/issues/22 which could be assisted with by https://github.com/dandi/dandischema/issues/21 ;-)

**Comment by yarikoptic** [comment]
Green! lets fix the world

**Pull request merged**